### PR TITLE
Feature/store_games

### LINKED ISSUE #23: [1pt] All data should be preserved by saving collections in .json files.
- All data should be preserved by saving collections in [file_name].json files.

### PR DESCRIPTION
**Pull request by Bondok6**
# Hello there 🙋‍♂️,

### Thanks for taking the time to review our project.

<hr>

## Changes according to requirements 🧾

- All data should be preserved by saving collections in [games].json files.
- Create `store` folder contains `game_read.rb` and `games_store.rb`.

Gif found on CodeReviewGifs.com hosted on Giphy.com

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -67,14 +69,15 @@ def add_game
     new_game.add_author(@authors[author_chosen - 1])
     @games << new_game
     puts 'The game has been added successfully!'
+    save_games(@games)
```
Comment (eduardosancho) [inline app.rb:72]: I see that you're saving the games everytime that a new one is added.
It seems definitely better than saving the data when the app is quit. Great job here! 🍔

File: store/game_read.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require_relative '../classes/game'
+require 'json'
+
+def read_games
+  data = []
+  if File.exist?('games.json')
```
Comment (eduardosancho) [inline store/game_read.rb]: You forgot to update this path :)
```suggestion
  if File.exist?('data/games.json')
```

**Review by eduardosancho — APPROVED** [review]
Well done!

I tested your changes in my terminal and no errors were thrown.
However, you forgot to update a file path in the `game_read`.
I added a suggestion to fix it.

Once that's done, it's good for merging. Congrats!

File: store/game_read.rb (hunk 2 above)
Comment (Bondok6) [reply store/game_read.rb]: Hi  @eduardosancho ,
oh, my bad. thank you for your review.

**Pull request merged**